boards/bluepill/include/board.h

```diff
@@ -46,6 +46,18 @@ extern "C" {
 #define LED0_TOGGLE         (LED0_PORT->ODR  ^= LED0_MASK)
 /** @} */
 
+/**
+ * @name    Default configuration in case a CC1101 is connected
+ * @{
+ */
+#define CC110X_PARAM_CS            GPIO_PIN(PORT_A, 4)
+#define CC110X_PARAM_GDO0          GPIO_PIN(PORT_B, 0)
+#define CC110X_PARAM_GDO2          GPIO_PIN(PORT_B, 1)
+#define CC110X_PARAM_PATABLE       (&cc110x_patable_433mhz)
+#define CC110X_PARAM_CONFIG        (&cc110x_config_433mhz_250kbps_300khz)
+#define CC110X_PARAM_CHANNELS      (&cc110x_chanmap_433mhz_300khz)
```

Comment:
That argument could cause a whole bunch of default config clutter within each `board.h`. Because why not add a default config for every WiFi/Ethernet/... shield there is for each arduino-compatible board?. Also to consider: the PINs you have chosen are basically arbitrary.

Reply:
(i.e. I'd rather like to discuss this in a separate PR / issue and not piggy-back it with this PR)

Reply:
> That argument could cause a whole bunch of default config clutter within each board.h. 

Agreed. I'll remove that